Dockerfile

```diff
@@ -3,18 +3,35 @@ FROM elifesciences/peerscout_client:${commit} AS client
 FROM elifesciences/python:cacd250ec201feab491c0738f261561b5360997b
 
 USER elife
-ENV PROJECT_FOLDER=/srv/peerscout
-RUN mkdir ${PROJECT_FOLDER}
-WORKDIR ${PROJECT_FOLDER}
```

Comment:
`PROJECT_FOLDER` is an [eLife convention](https://github.com/search?q=org%3Aelifesciences+PROJECT_FOLDER&type=Code)

Reply:
some [steps](https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/0ced62719af287aa84eeb73380d4ccf86573973d/vars/dockerProjectTests.txt) (that are not used here probably) rely on that

Reply:
> `PROJECT_FOLDER` is an [eLife convention](https://github.com/search?q=org%3Aelifesciences+PROJECT_FOLDER&type=Code)

I didn't realise that. I used `PROJECT_HOME` in [all of my other projects](https://github.com/search?p=1&q=org%3Aelifesciences+PROJECT_HOME&type=Code). But seem to be outnumbered by your search result. None of them made use of `dockerProjectTests` though.

Reply:
If only there was a standard Docker image label for that...